clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy

```diff
@@ -177,6 +179,39 @@ class KubernetesApiAdaptor {
     client.namespaces().create(namespace)
   }
 
+  static KubernetesLoadBalancerDescription fromService(Service service) {
+    if (!service) {
+      return null
+    }
+
+    def loadBalancerDescription = new KubernetesLoadBalancerDescription()
+
+    loadBalancerDescription.name = service.metadata.name
+    def parse = Names.parseName(loadBalancerDescription.name)
+    loadBalancerDescription.app = parse.app
+    loadBalancerDescription.stack = parse.stack
+    loadBalancerDescription.detail = parse.detail
+    loadBalancerDescription.namespace = service.metadata.namespace
+
+    loadBalancerDescription.clusterIp = service.spec.clusterIP
+    loadBalancerDescription.loadBalancerIp = service.spec.loadBalancerIP
+    loadBalancerDescription.sessionAffinity = service.spec.sessionAffinity
+    loadBalancerDescription.serviceType = service.spec.type
+
+    loadBalancerDescription.externalIps = service.spec.externalIPs ?: []
+    loadBalancerDescription.ports = service.spec.ports?.collect { port ->
+      new KubernetesNamedServicePort(
+          name: port.name,
+          protocol: port.protocol,
+          port: port.port ?: 0,
+          targetPort: port.targetPort?.intVal ?: 0,
+          nodePort: port.nodePort ?: 0
```

Comment:
Since these are just declared as `int`s, they'll default to 0 without the need of the elvis operator